cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -471,14 +468,14 @@
               </list>
             </t>
             <t>
-              If the COSE Header contains a "content type" header value
-              corresponding to the media type "application/cwt", then
-              the Message is a CWT that was the subject of nested signing
-              or encryption operations.  In this case, return to Step 1,
+              If the Message begins with a COSE CBOR tag, then
+              the Message is a CWT that was the subject of nested signing,
+              MACing or encryption operations.  In this case, return to Step 1,
               using the Message as the CWT.
             </t>
             <t>
-              Verify that the Message is a valid CBOR object; let the CWT Claims Set be this CBOR object.
+              Verify that the Message is a valid CBOR object; let the CWT
+              Claims Set be this CBOR object.
```

Comment:
Please remove this extra line break, as breaking in the middle of "CWT Claims Set" makes searching and editing the source harder.